### ISSUE TITLE
Neither dejavu nor terminus is used

### ISSUE THREAD

**Issue by juancholopes** [vinceliuice/Elegant-grub2-themes#5]
When I install the theme, the corresponding fonts are not installed or rather they are not applied.

**Comment by vinceliuice** [comment]
Changed the font to unifont

**Comment by Firestar-Reimu** [comment]
I rather like Dejavu Sans font than monospaced fonts when display the OS name, how to change?

**Comment by vinceliuice** [comment]
@Firestar-Reimu  You can install old version: https://github.com/vinceliuice/Elegant-grub2-themes/releases/tag/2024-10-23

**Comment by Firestar-Reimu** [comment]
I edited the theme.txt and added dejavu sans from the older release

and I see that: 

- the **Boot menu** is dejavu sans, which is what I want
- the terminal saying `**Loading Linux linux, Loading initial ramdisk**` is also dejavu sans, which is **not** what I want, I want it to be **monospaced**

But if I change to `terminal-font: "Terminus Regular 18"`, then GRUB menu will also use terminus instead of dejavu

---

Your other theme, https://github.com/vinceliuice/grub2-themes/releases/tag/2024-08-19 has no this problem, it can use dejavu in the GRUB menu and terminus in the terminal

```
# GRUB2 gfxmenu Linux theme
# Designed for any resolution

# Global Property
title-text: ""
desktop-image: "background.jpg"
desktop-color: "#f0f0f0"
terminal-font: "Terminus Regular 18"
terminal-box: "terminal_box_*.png"
terminal-left: "0"
terminal-top: "0"
terminal-width: "100%"
terminal-height: "100%"
terminal-border: "0"

# Show the boot menu
+ boot_menu {
  left = 52%
  top = 18%
  width = 28%
  height = 54%
  item_font = "Dejavu Sans Regular 32"
  item_color = "#333333"
  selected_item_color = "#ffffff"
  icon_width = 64
  icon_height = 64
  item_icon_space = 12
  item_height = 96
  item_padding = 6
  item_spacing = 12
  selected_item_pixmap_style = "select_*.png"
}

+ image {
  top = 0%
  left = 0%
  file = "info.png"
}

+ image {
  top = 29%
  left = 22%
  file = "logo.png"
}

# Show a countdown message using the label component
+ label {
  top = 76%
  left = 55%
  width = 23%
  align = "center"
  id = "__timeout__"
  text = "Booting in %d seconds"
  color = "#333333"
  font = "Dejavu Sans Regular 32"
}
```

**Comment by vinceliuice** [comment]
Maybe it lacks terminal_box*.png files

**Comment by vinceliuice** [comment]
<img width="2298" height="1044" alt="Image" src="https://github.com/user-attachments/assets/350fedd7-6060-41c7-adc9-8ca77f417bd4" />

you can copy it to common folder

**Comment by Firestar-Reimu** [comment]
No, the terminal saying **Loading Linux linux, Loading initial ramdisk** is still dejavu sans, which is not what I want, I want it to be monospaced


terminal-font: "Terminus Regular 18"
item_font = "Dejavu Sans Regular 32"

> Maybe it lacks terminal_box*.png files
it just added a outline box per grub item
